SIPs/SIP-000X.md

```diff
@@ -0,0 +1,164 @@
+
+## Purpose
+The purpose of the Sovryn Improvement Proposal Process ("the Sovryn proposal process") is to provide a structured process for making changes to the shared resources of Sovryn. For these shared resources, a governance process is needed to grant or deny access and approve or reject proposed changes. By creating a fair, lightweight, and transparent governance process, the SIP-X authors hope to give SOV holders a meaningful say in the governance of Sovryn and increase the chances of Sovryn's success.
+
+## Proposal workflow
+Parties directly involved in the process are the _SIP author(s)_ (you), _Sovryn Voters_, _Sovryn Guardians_, and _SIP Editors_. Each of these roles is explained later in this document.
+
+Proposals SHOULD follow this workflow:
+
+* Stage I: Select SIP Track  
+* Stage II: Pre-proposal  
+* Stage III: Draft Proposal  
+* Stage IV: Review Periods  
+* Stage V: Sovryn DAO Vote  
+
+At a high level, the SIP-X workflow looks like this:
+
+TODO: Upload and link workflow image here
+
+> A proposal does not have to go through Stages I-IV before it is submitted for a Sovryn Vote in Stage V. However it is strongly RECOMMENDED to follow this whole process so that the proposal gets a proper review and is not perceived with suspicion or hostility by Sovryn Voters. If a proposal is urgent, it MAY go through an accelerated review process if needed; if you feel your proposal is urgent, you SHOULD make this urgency and its reason known when you introduce your proposal to the community to align expectations.
+
+### Stage I: Select SIP Track
+Before you spend time working on a proposal, you SHOULD make sure the proposal complies with SIP-X and has a chance of passing review by your peers. Review the SIP tracks and their requirements, then select the track that you think is best for your proposal. If your proposal meets the requirements, it has a much greater chance of being approved by Sovryn Voters.
+
+There are four tracks that a SIP can be categorized into. Select the one you think is best for your SIP:
+
+* Contract: proposals for modifying one or more Sovryn smart contracts  
+* Finance: proposals for transferring funds from the Sovryn Treasury  
+* Meta: proposals for changing SIP-0 or SIP-X (“changing the way things are changed”)  
+* Proclamations: proposals for making a public statement on behalf of Sovryn  
+
+A proposal may be categorized as “Other” until a new, appropriate track is approved as part of a Meta SIP.
+
+In addition to the requirement that all SIPs must be consistent with SIP-0, each track has its own requirements for SIPs as follows:
+
+**Contract**  
+Proposals made to the Contract track MUST change one or more of the Sovryn smart contracts. The proposal MUST link to the existing contract(s) to be modified and to the source code of the proposed contract modification(s). All Contract track proposals MUST include a link to a third party audit report of the specified modification(s).
```

Comment:
I agree that all of our code must be audited by a third party. We've got a third party involved on an ongoing basis. All of our PRs are getting reviewed before they are deployed. At this moment, we do not have audit reports for these ongoing reviews, so we can't add a link to one. Should it suffice to send a link to the PRs, so people can verify that the changes have been sufficiently reviewed and approved?

Reply:
Why don't we have audit reports?

Reply:
i guess it's to keep the overhead low. if it's needed we can start requesting a report for each review. i'm just stating the current circumstances.

Reply:
> Should it suffice to send a link to the PRs, so people can verify that the changes have been sufficiently reviewed and approved?

Ultimately it's up to each voter to decide how much or how little information they want to see in each proposal. Right now there's not really a way to enforce this process, only to strongly suggest that it be followed. So while the intention here is that each Contract proposal have a proper audit report, voters may adjust their expectations for certain proposals and we can't force anything otherwise.

If it helps contextualize this requirement, the reason I propose a formal audit report is that I expect contract changes to be few and far between once the system gets to a certain point of stability. So while repos may have a lot of activity, changes won't be merged onchain right away but could/would be applied in batches. With this, having a proper audit report for each Contract proposal shouldn't be too much overhead. Open to suggested changes here though, as this is a bit outside my area of expertise!